plugins/modules/ovirt_host.py

```diff
@@ -613,7 +613,7 @@ def main():
             ret = hosts_module.action(
                 action='upgrade',
                 action_condition=lambda h: h.update_available,
-                wait_condition=lambda h: not h.update_available or h.status == result_state and (
+                wait_condition=lambda h: not h.update_available and h.status == result_state and (
```

Comment:
Is the update_availible False even when the upgrade fails? 
If it would be still available the module would still wait and timeout.

Reply:
I should check.
But even then, the status will never be result_state (but FailedInstall), so it will always wait and timeout no?

Reply:
Better logic in the next commit. That should cover the cases like before.